### ISSUE TITLE
function-type false positive for array constant

### ISSUE THREAD

**Issue by mathilde-pellerin** [Gillespie59/eslint-plugin-angular#382]
Hi,

I just upgrade my project to `"eslint-plugin-angular": "^1.0.0"` and now ESlint complains about array constants : 
```
angular.module('mymodule.constants', [])
    .constant('URL_WHITELIST', [
      'self',
      'https://someUrl/**'
    ])
```

> error  Use anonymous functions instead of named function  angular/function-type

The rule definition I use:
`"angular/function-type": [2,  "anonymous"],`

but it's the same if I use following rule instead
`"angular/function-type": [2,  "named"],`
